app/src/main/res/layout/activity_main.xml

```diff
@@ -1,18 +1,18 @@
 <?xml version="1.0" encoding="utf-8"?>
-<android.support.constraint.ConstraintLayout
+<androidx.constraintlayout.widget.ConstraintLayout
         xmlns:android="http://schemas.android.com/apk/res/android"
         xmlns:tools="http://schemas.android.com/tools"
         xmlns:app="http://schemas.android.com/apk/res-auto"
         android:layout_width="match_parent"
         android:layout_height="match_parent"
         tools:context=".MainActivity">
-    <TextView
-            android:layout_width="wrap_content"
-            android:layout_height="wrap_content"
-            android:text="My second commit!"
+    <androidx.recyclerview.widget.RecyclerView
+            android:id="@+id/dogRV"
+            android:layout_width="match_parent"
+            android:layout_height="match_parent"
```

Comment:
I ran the app and I saw that each row is as tall as the screen. I think there is something off with the height of your layout. Either this needs to be a `wrap_content` or something else related to this. I've had this problem in the past. It's just a matter of tweaking the right attribute. You can probably find out what the change by googleing for it.